server/handlers/component_handler.go

```diff
@@ -62,7 +62,7 @@ func (h *Handler) GetMeshmodelModelsByCategories(rw http.ResponseWriter, r *http
 	}
 	if r.URL.Query().Get("search") != "" {
 		filter.Greedy = true
-		filter.Name = r.URL.Query().Get("search")
+		filter.DisplayName = r.URL.Query().Get("search")
```

Comment:
@Abhishek-kumar09 this works well with MeshMap, or no?

Reply:
@abdullah1308 maybe you can confirm.

Reply:
@leecalcote I've checked out the meshmap code, these changes should work well with Meshmap.

Reply:
The endpoints changed in this PR aren't being used anywhere. The changes made are to make the search functionality in these endpoints inline with the endpoints that are being used.